docs/LDES.md

```diff
@@ -0,0 +1,111 @@
+# LDES in LMB
+
+> [!CAUTION]
+> This section contains the specification of how we will be using LDES. This is not implemented yet and unproven. A PoC implementation will be created in the sprint of 26-02 -> 08-03
+
+## LDES Feeds Produced by LMB
+
+We will be using LDES to sync with different partners:
+
+- the Meeting notes system (e.g. Gelinkt Notuleren, but it could be a vendor specific provider too, for easy writing the text below will use GN to denote any software taking the role of the Notulering-system)
+- the Vlaamse Mandatendatabank
+
+### Draft Mandataris instances for GN
```

Comment:
I'm missing some context about when these draft mandatarissen will be created, is this on request or ...